addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -465,6 +464,29 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasDescriptionArea() {
+            return Boolean(
+                this.thread &&
+                (this.thread.description ||
+                this.thread.isChannelDescriptionChangeable)
```

Comment:
lint: wrong format for multi-line, should be one condition per line, and matching indent for parenthesis. Or change to single line.